src/Transports.AspNetCore.NewtonsoftJson/InternalGraphQLRequest.cs

```diff
@@ -0,0 +1,21 @@
+using GraphQL.Server.Common;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace GraphQL.Server.Transports.AspNetCore.NewtonsoftJson
+{
+    /// <summary>
+    /// A type for deserializing directly into that suits the System.Text.Json serializer.
+    /// </summary>
+    public class InternalGraphQLRequest
```

Comment:
So... maybe do `InternalGraphQLRequest` internal ?

Reply:
Whoops, had updated the other one but not this one. Fixed.